org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/license/LicenseCheckedList.java

```diff
@@ -386,299 +386,299 @@ private static Provider getProviderFromId(final Providers providers, final Strin
         return provider;
     }
 
-	private class ArchiveSaxHandler extends AbstractSaxHandler {
-		
-	    private static final String ELEMENT_ARCHIVES = "archives";
```

Comment:
Also, could you please revert indentation changes?

Reply:
We should agree an a unique formatting and enforce it by providing a eclipse formatter rules file. 
For changing the indentation, I actually used the A3k formatting rules that were used so far.